### ISSUE TITLE
Add "Bought" or "Audible Plus" flag

### ISSUE THREAD

**Issue by fsafq41** [rmcrackan/Libation#756]
**Is your feature request related to a problem? Please describe.**
No, just something that would make it easier to sort and work with existing library

**Describe the solution you'd like**
Audible knows whether a book has been bought or is borrowed from the Audible Plus library. If such flag is available from their API, please could you add this tag to sort or filter on it

**Describe alternatives you've considered**
The Audible app, but I'm avoiding that

**Additional context**
None

**Comment by RandmTask** [comment]
That would be fantastic, and be able to filter and sort by Plus or Bought

**Comment by ElysiumFic** [comment]
I agree this would be a handy feature, particularly since I prefer to stream my Audible Plus files directly from Audible so the author gets paid.

**Comment by rmcrackan** [comment]
Several audible plus features have been added since this request. Plz see if any of them fit your needs.
